actnow/accounts/models.py

```diff
@@ -81,3 +82,12 @@ class ActNowUser(AbstractBaseUser, PermissionsMixin, TimestampedModelMixin):
     class Meta:
         verbose_name = _("user")
         verbose_name_plural = _("users")
+
+
+class ActNowApplication(AbstractApplication):
+    user = models.OneToOneField(
```

Comment:
```suggestion
class Application(AbstractApplication):
    owner = models.OneToOneField(
```

I think it's not cool using brand names like ActNow _**in**_ code since it can easily change.

Reply:
Changing the `user` field to `owner` will result in 2 user fields(one being called `owner`) because we already have another `user` field [here](https://github.com/jazzband/django-oauth-toolkit/blob/master/oauth2_provider/models.py#L77) 
Adding our `user` ensures an application is only associated with one `user`

Reply:
weird but what are you going to do 🤷🏾‍♂️